ndmg/graph/gen_graph.py

```diff
@@ -196,7 +196,7 @@ def make_graph(self, error_margin=2, overlap_thr=1, voxel_size=2):
         mx = len(np.unique(self.rois.astype(np.int64))) - 1
         self.g = nx.Graph(ecount=0, vcount=mx)
         edge_dict = defaultdict(int)
-        node_dict = dict(zip(np.unique(self.rois) + 1, np.arange(mx) + 1))
+        node_dict = dict(zip(np.unique(self.rois).astype('int16') + 1, np.arange(mx) + 1))
```

Comment:
Why do we need this to be `int16`?

Reply:
consistency. Not all parcellations/atlases use integer intensities, and this imposes that as a standard.